src/RecipeCard.tsx

```diff
@@ -0,0 +1,39 @@
+import './RecipeCard.css';
+
+type RecipeProps = {
+  recipe: {
+    id: number;
+    name: string;
+    difficulty: string;
+    tags: string[];
+    image: string;
+  };
+};
+
+const RecipeCard = ({ recipe }: RecipeProps) => {
```

Comment:
Props가 길지 않다면 이런 식으로 작성해보는 것도 좋을 것 같습니다!

```suggestion
const RecipeCard = ({ recipe }: { recipe: RecipeBrief }) => {
```